examples/routing_service/routing_service_file_adapter/c/README.md

```diff
@@ -13,21 +13,43 @@ adapter will take care of creating a file for every stream received. The files
 will be created on a specific path, which you can also configure in the XML
 configuration file.
 
-## Building C example
+## Building the C Example
 
 In order to build this example, you need to provide the following variables to
 `CMake`:
 
-- `CONNEXTDDS_DIR`
-- `CONNEXTDDS_ARCH` (needed only if you have multiple architectures installed)
+- `CMAKE_BUILD_TYPE`: specifies the build mode. Valid values are Release and 
+  Debug
+- `BUILD_SHARED_LIBS`: whether to build the library as a shared library or not.
+  This should be set to ON when loading the plugin from the Routing Service
+  executable. However, Routing Service supports loading of static libraries when
+  using the service as a library. See note below.
 
 ```bash
 $mkdir build
-$cmake -DCONNEXTDDS_DIR=<Connext DDS Directory>
-    -DCONNEXTDDS_ARCH=<Connext DDS Architecture>
-    -DBUILD_SHARED_LIBS=ON|OFF
-    -DCMAKE_BUILD_TYPE=Debug|Release ..
-cmake --build .
+$cmake -DBUILD_SHARED_LIBS=ON|OFF -DCMAKE_BUILD_TYPE=Debug|Release ..
+$cmake --build .
```

Comment:
can you remove the `$` character from the beginning of the line of the README's code you modified in this PR to have consistent documentation across the languages?

Reply:
Sure thing, let me review this in all the README files, in fact.